src/components/Avatar.tsx

```diff
@@ -0,0 +1,25 @@
+import "./Avatar.scss";
+
+import React from "react";
+
+type AvatarProps = {
+  children: React.ReactNode;
+  className?: string;
+  onClick: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
+  color: string;
+};
+
+export const Avatar = ({
```

Comment:
Let's take `title` prop and display `Unnamed collaborator` on hover if user doesn't supply username.

Reply:
Yeah hover sounds good - showing either full username or unnamed.

Reply:
Added tooltip component which wraps around the avatars. Title is a weird name for the prop - changed children to enforce string-only but happy to make the prop 'initials' or something.